js/packages/cli/src/commands/verifyTokenMetadata/index.ts

```diff
@@ -90,7 +90,9 @@ export const verifyCreatorCollation = (
 };
 
 export const verifyImageURL = (image, files, manifestFile) => {
-  const expectedImagePath = `image${EXTENSION_PNG}`;
+  // The image is expected to have the same name as the index
+  const fileIndex = manifestFile.split('/').pop().split('.')[0];
```

Comment:
you can pass tokenMetadata to this function from verifyMetadataManifests and use tokenMetadata.image